From 454923709a1a1e480554c400e053aea9a1ba951a Mon Sep 17 00:00:00 2001
From: Akira TAGOH <akira@tagoh.org>
Date: Thu, 10 May 2018 22:01:29 +0900
Subject: [PATCH] Change the emboldening logic again

enable emboldening when request was >= bold and font was <= medium

https://bugs.freedesktop.org/show_bug.cgi?id=106460
---
 conf.d/90-synthetic.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/conf.d/90-synthetic.conf b/conf.d/90-synthetic.conf
index e344e4a..6b929dd 100644
--- a/conf.d/90-synthetic.conf
+++ b/conf.d/90-synthetic.conf
@@ -42,9 +42,9 @@
  -->
 
 	<match target="font">
-		<!-- check to see if the font is just regular -->
+		<!-- check to see if the weight in the font is less than medium which possibly need emboldening -->
 		<test name="weight" compare="less_eq">
-			<const>regular</const>
+			<const>medium</const>
 		</test>
 		<!-- check to see if the pattern requests bold -->
 		<test target="pattern" name="weight" compare="more_eq">
-- 
2.17.0

